MANIFEST.in

```diff
@@ -1,6 +1,9 @@
-# Source distribution files.
-include README.md
-include setup.py
+include README.rst LICENSE tox.ini .travis.yml docs/Makefile manage.py
```

Comment:
- We aren't using tox right now but it is on my list but I will leave it out for now.
- I'll remove the travis file
- setup.cfg gets included by default along with setup.py
- you're right, we aren't currently using a tests directory
- Having the raw rst files in the dist package seems helpful but I don't think I'd want to include any potentially large binary files like images that may be part of the docs.